content/shmem_reductions.tex

```diff
@@ -257,6 +257,8 @@ \subsubsubsection{PROD}
     \VAR{nreduce} must be of type integer.}
 
 \begin{DeprecateBlock}
+\apiargument{IN}{nreduce}{In active-set based \ac{API} calls,
```

Comment:
I can't comment on the above line but should we change the above sentence from:
```
\apiargument{IN}{nreduce}{The number of elements in the \dest{} and \source{}
    arrays. In teams based \ac{API} calls, \VAR{nreduce} must be of type size\_t.
    In deprecated active-set based \ac{API} calls,
    \VAR{nreduce} must be of type integer.}
```
to
```
\apiargument{IN}{nreduce}{The number of elements in the \dest{} and \source{}
    arrays. In teams based \ac{API} calls, \VAR{nreduce} must be of type size\_t.}
```

Reply:
Nice catch